Nautilus/Documentation/guides/assetbundles.md

```diff
@@ -0,0 +1,114 @@
+Here you can either assign your prefab to an existing AssetBundle or create a new one.  
+![assetbundles-assigning](../images/guides/assetbundles-assigning.png)
+
+Once you've assigned your AssetBundle, click ``Window``, and then ``AssetBundleBrowser``.  
+Here you can see all the assets that will be put into your AssetBundle.  
+![assetbundles-browser](../images/guides/assetbundles-browser.png)
+
+Click on the build tab, and change the ``Build Target`` to "Standalone Windows" if it isn't already.  
+You can also configure the ``Output Path`` of the build.  
+After all that is set, click ``Build``.
+
+Once your AssetBundle is built, move the built file to the location of your mod in the Plugins folder.  
+I prefer to put them inside a folder namded "Assets", but that isn't necessary.  
+![assetbundles-assets-folder](../images/guides/assetbundles-assets-folder.png)
+
+Now the hard part is done! All you need to do is load it into your mod's code.  
+To do this, you will need to add an extra reference to your mod.  
+Right click ``References`` in Visual Studio under your project and select add reference.
+
+Click Browse, and navigate to ``[Game Location]/Subnautica_Data/Managed``, and add ``UnityEngine.AssetBundleModule.dll`` as a reference.  
+This will allow you to load your AssetBundle into code.  
+![assetbundles-references](../images/guides/assetbundles-references.png)
+
+Here's some example code of how you can load in your prefab:
+```csharp
+using BepInEx;
+using System.IO;
+using System.Reflection;
+using UnityEngine;
+
+namespace Examples
+{
+    internal class AssetBundles : BaseUnityPlugin
+    {
+        //I usually do this in my Plugin script but technically you can do it wherever
+        public static AssetBundle assetBundle { get; private set; }
+
+        //This gets the path to the "Assets" folder inside my plugin folder
+        //If you don't have an assets folder you can replace "AssetsFolderPath" with Assembly.GetExecutingAssembly().Location
+        //That just gets the path to the .dll of the mod
+        public static string AssetsFolderPath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "Assets");
+
+        private void Awake()
+        {
+            //Keep in mind that the assetbundle can only be open in one place at a time, so keep a reference
+            assetBundle = AssetBundle.LoadFromFile(Path.Combine(AssetsFolderPath, "mirrorassetbundle"));
+
+            //This name needs to be the exact same name as the prefab you put in the bundle
+            GameObject mirrorVariant1 = assetBundle.LoadAsset<GameObject>("Mirror_Variant1");
+        }
+    }
+}
+```
+
+And just like that you have your prefab in your code!  
+To use this in a Nautilus prefab, just use
+```csharp
+myCustomPrefab.SetGameObject(myAssetBundleGO);
+```
+Instead of using a CloneTemplate.
```

Comment:
```suggestion
instead of using a prefab template. For more complicated prefabs (such as buildables), you can create your own method that loads the prefab from the asset bundle and applies extensive modifications to it. You can *also* apply changes to the prefab directly, e.g., `myAssetBundleGO.AddComponent<WorldForces>();`
```